tools/upgrader.py

```diff
@@ -0,0 +1,37 @@
+import os
+import argparse
+
+
+def upgrade(frm,to):
+
+    if frm == "2.4.0" and to == "2.5.0":
+        print(f"UPGRADING FROM {frm} TO {to} ...")
+
+        os.system("curl -O https://raw.githubusercontent.com/IBM/ibm-spectrum-scale-csi/v2.5.0/generated/installer/ibm-spectrum-scale-csi-operator.yaml")
+        os.system("kubectl apply -f ibm-spectrum-scale-csi-operator.yaml")
+        os.system("kubectl get pod -n ibm-spectrum-scale-csi-driver")
+        os.system("kubectl describe pod ibm-spectrum-scale-csi-operator-8947b76cb-k8ggx -n ibm-spectrum-scale-csi-driver")
+
+    elif frm == "2.4.0" and to == "2.5.1":
+        print(f"UPGRADING FROM {frm} TO {to} ...")
+
+        os.system("curl -O https://raw.githubusercontent.com/IBM/ibm-spectrum-scale-csi/v2.5.1/generated/installer/ibm-spectrum-scale-csi-operator.yaml")
+        os.system("kubectl apply -f ibm-spectrum-scale-csi-operator.yaml")
+        os.system("kubectl get pod -n ibm-spectrum-scale-csi-driver")
+        os.system("kubectl describe pod ibm-spectrum-scale-csi-operator-8947b76cb-k8ggx -n ibm-spectrum-scale-csi-driver")
```

Comment:
The 4 os.system calls are identical in the two scenarios you created, you could make a function for it and pass in the "to" version, which determines the tag of the operator yaml to curl..

Then potentially this code could work for any upgrade since there's no checking on the the "from" version, it's just printing a msg.... but downloading the target version of the operator yaml